### ISSUE TITLE
[Rendering] ACROverflow Target should send whether the actions are from RootLevel of the adaptive Card

### ISSUE THREAD

**Issue by shabripabba** [microsoft/AdaptiveCards#6459]
### Target Platforms

iOS

### SDK Version

2.8.0-beta3

### Application Name

Microsoft Teams

### Problem Description

When a Root level Overflow button is tapped, then ACROverflowTarget sends a delegate call to `[_rootView.acrActionDelegate onDisplayOverflowActionMenu:_menuItems alertController:_alert]` but there are scenarios where the client will need to know whether the target sending the delegate message isAtRootLevel, so that Client can do something with that information.

### Card JSON

```JSON
{
  "body": [
    {
      "size": "large",
      "weight": "bolder",
      "type": "TextBlock",
      "text": "Overflow Action Test Card"
    },
    {
      "type": "TextBlock",
      "text": "ActionSet **all secondary actions**"
    },
    {
      "actions": [
        {
          "data": {
            "key": "Submit from overflow menu"
          },
          "title": "Action Submit",
          "type": "Action.Submit",
          "mode": "secondary"
        },
        {
          "mode": "secondary",
          "title": "OpenUrl",
          "type": "Action.OpenUrl",
          "url": "https:\/\/github.com\/Microsoft\/AdaptiveCards",
          "iconUrl": "https:\/\/urlp.asm.skype.com\/v1\/url\/content?url=https%3a%2f%2fwww.clipartmax.com%2fpng%2ffull%2f211-2118815_file-novosel-mushroom-svg-mario-mushroom-icon.png"
        },
        {
          "title": "Action.ShowCard",
          "card": {
            "type": "AdaptiveCard",
            "actions": [
              {
                "title": "Neat!",
                "type": "Action.Submit"
              }
            ],
            "body": [
              {
                "type": "TextBlock",
                "text": "What do you think?"
              }
            ]
          },
          "mode": "secondary",
          "type": "Action.ShowCard"
        }
      ],
      "type": "ActionSet"
    },
    {
      "type": "TextBlock",
      "text": "ActionSet **primary + secondary actions**"
    },
    {
      "actions": [
        {
          "title": "View",
          "data": {
            "key": "View"
          },
          "type": "Action.Submit"
        },
        {
          "data": {
            "key": "Edit"
          },
          "title": "Edit",
          "type": "Action.Submit",
          "mode": "secondary"
        },
        {
          "data": {
            "key": "Delete"
          },
          "title": "Delete",
          "type": "Action.Submit",
          "mode": "secondary"
        }
      ],
      "type": "ActionSet"
    },
    {
      "type": "TextBlock",
      "text": "Check actions.length > maxActions"
    },
    {
      "actions": [
        {
          "title": "Action 1",
          "type": "Action.Submit"
        },
        {
          "title": "Action 2",
          "type": "Action.Submit"
        },
        {
          "title": "Action 3",
          "type": "Action.Submit"
        },
        {
          "title": "Action 4",
          "type": "Action.Submit"
        },
        {
          "title": "Action 5",
          "type": "Action.Submit"
        },
        {
          "title": "Action 6",
          "type": "Action.Submit"
        },
        {
          "title": "Action 7",
          "type": "Action.Submit"
        },
        {
          "title": "Action 8",
          "type": "Action.Submit"
        },
        {
          "title": "Action 9",
          "type": "Action.Submit"
        }
      ],
      "type": "ActionSet"
    },
    {
      "type": "TextBlock",
      "text": "========= I am bottom line of body ========="
    }
  ],
  "type": "AdaptiveCard",
  "actions": [
    {
      "url": "https:\/\/adaptivecards.io",
      "title": "OpenUrl 1",
      "type": "Action.OpenUrl"
    },
    {
      "data": {
        "key": "Submit from overflow menu"
      },
      "title": "Action Submit",
      "type": "Action.Submit",
      "mode": "secondary"
    },
    {
      "mode": "secondary",
      "title": "OpenUrl",
      "type": "Action.OpenUrl",
      "url": "https:\/\/github.com\/Microsoft\/AdaptiveCards",
      "iconUrl": "https:\/\/urlp.asm.skype.com\/v1\/url\/content?url=https%3a%2f%2fwww.clipartmax.com%2fpng%2ffull%2f211-2118815_file-novosel-mushroom-svg-mario-mushroom-icon.png"
    },
    {
      "title": "Action.ShowCard",
      "card": {
        "type": "AdaptiveCard",
        "actions": [
          {
            "title": "Neat!",
            "type": "Action.Submit"
          }
        ],
        "body": [
          {
            "type": "TextBlock",
            "text": "What do you think?"
          }
        ]
      },
      "mode": "secondary",
      "type": "Action.ShowCard"
    }
  ],
  "version": "1.5"
}
```

**Comment by jonmill** [comment]
Triage - Shabri has a fix ready to go for iOS here, if this is approved

**Comment by RebeccaAnne** [comment]
@shabripabba This is approved, please feel free to send a PR. Thanks!

**Comment by shabripabba** [comment]
Merged into latest main and will be available in 2.8.0
